bin/test-example

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
+
+if [ $# -eq 0 ]; then
+    echo "usage: $0 /path/to/exercise/example"
+    exit 1
+fi
+
+exampledir=$(cd "$1" && pwd)
+
+examplename=$(basename "$exampledir")
+exercisedir=$(cd "$exampledir/../.." && pwd)
+exercisename=$(basename "$exercisedir")
+xhaskell="$(cd "$(dirname "$0")/.." && pwd)"
+
+buildfolder="${xhaskell}/build/${exercisename}/${examplename}"
+mkdir -p "${buildfolder}"
+cleanup() {
+    rm -r "${xhaskell}/build"
```

Comment:
* If remove just `buildfolder`, then `build/$exercisename` will remain for each exercise, which isn't a good cleanup. So I remove all of `build`.